doc/dev/GETTING_STARTED.md

```diff
@@ -0,0 +1,159 @@
+# Getting started guide
+
+This is a getting started guide, aimed to people that are not used
+to C++ programming, configuring and compiling. If you are already used
+to such processes, you may be looking for our build guide.
+
+This guide will help you to compile your own version of F3D with required dependencies.
+Optional dependencies or covered in the build guide.
+
+ - If you are using [Linux]
+ - If you are using [Windows]
+ - If you are using [macOS]
+
+## Linux
+
+Note: The following steps were tested with Ubuntu 22 and ArchLinux but
+should work for other OSes as stated, as long as listed packages are available.
+
+### Install dependencies
+
+First make sure your system is up-to-date,
+then install the required dependencies using the terminal.
+
+#### Debian/Ubuntu/Mint
+
+```
+sudo apt install build-essential git git-lfs cmake libvtk9-dev
+```
+
+#### Fedora/Centos/RedHat
+
+```
+sudo yum install make automake gcc gcc-c++ kernel-devel git git-lfs cmake vtk
+```
+
+#### Arch Linux
+
+```
+sudo pacman -S base-devel git git-lfs cmake vtk
+```
+
+### Compile F3D
+
+Open a terminal.
+
+```
+mkdir f3d
+cd f3d
+mkdir build
+git clone https://github.com/f3d-app/f3d src
+cd build
+cmake ../src
+make
+```
+
+You can then simply run F3D from the command line:
+
+```
+./bin/f3d
+```
+
+## Windows
+
+Note: The following steps were tested with Visual Studio Community 2022, but should also work for other versions
+
+#### Install Dependencies
+ * Download and install [git bash for windows][gitforwindows]
+ * Download and install [cmake][cmake-download]
+ * Download and install [Visual Studio 2022 Community Edition][visual-studio]
+
+#### Recover F3D sources
+
+ * Open git bash
+
+```sh
+cd C:
+mkdir dev
+cd f3d
+git clone https://github.com/f3d-app/f3d.git src
+mkdir build
+```
+
+#### Install dependencies using vcpkg
+
+ * Open git bash
+
+```sh
+cd C:/dev
+git clone https://github.com/Microsoft/vcpkg.git
+```
+
+ * Open cmd
+
+```sh
+cd C:\dev
+./vcpkg/bootstrap-vcpkg.sh
+cmake -B ./f3d/build -S ./f3d/src -DCMAKE_TOOLCHAIN_FILE=./vcpkg/scripts/buildsystems/vcpkg.cmake 
+```
+
+Note: Last command will take a while. It download, compile and install all dependencies, including
+many optional ones.
+
+#### Build F3D
+
+ * Open cmd
+
+```sh
+cd C:\dev\f3d\build
+cmake .
+cmake --build .
+```
+
+#### Run
+
+ * Double click on the `C:\dev\f3d\build\bin_Debug\f3d.exe` executable
+
+## macOS
+
+### Install brew
+
+Open a terminal.
+
+```
+/bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"
+```
+
+TODO Check if XCode is needed
```

Comment:
You don't need xcode. There is a terminal command to install developer tools

Reply:
actually needed, brew tells me:

`A full installation of XCode is needed to install to compile this software...`

when I run `brew install vtk`. Ill need to start from scratch, not enough space to install XCode

Reply:
Even if you are using Ninja generator?

Reply:
Can you change the generator used by brew ? In any case, I doubt that installing ninja is easier than installing xcode and a beginner user.

Reply:
You need xcode to install brew itself?

Reply:
I need either Command Line Tools or XCode to install brew.

I need XCode to install specific packages has their are flagged as such in brew system.